reports-backend/src/main/java/org/jboss/da/listings/impl/service/WhiteArtifactServiceImpl.java

```diff
@@ -48,6 +51,41 @@
     }
 
     @Override
+    public List<ProductVersion> toProductsContainingOnlyWhiteArtifacts(List<ProductVersion> products){
+    	
+    	// Get all version artifacts
+    	List<ProductVersion> productVersions = new ArrayList<>(products);
```

Comment:
Just a habit I guess - prevents having to assume the supplied (same as returned) list isn't manipulated externally to the method. Happy to change it either way.